meteor/meteor
# Convert profile-require.ts to TypeScript

Converted `profile-require.js` to TypeScript. This is a file that summarizes time spent "requiring" things during a run if the given environment env variable is set.

To test:
Set this env variable:
```sh
export METEOR_PROFILE_REQUIRE=true
```
Then run some Meteor app using your local Meteor version. You should see a list that looks like this (excerpted):
```
29.00 TOP
14.00 @babel/parser [via TOP:warehouse.js]
12.00 npm [via TOP:warehouse.js]
10.00 acorn [via TOP:warehouse.js]
8.00 sqlite3 [via TOP:warehouse.js]
7.00 npmlog [via TOP:warehouse.js]
...
TOTAL: ours 30.00, other 502.00, grand total 532.00
```
